src/core/QCheck2.ml

```diff
@@ -353,15 +361,20 @@ module Gen = struct
 
   let (--.) low high = float_range ?origin:None low high
 
-  let neg_int : int t = nat >|= Int.neg
+  let int_neg : int t = nat >|= Int.neg
 
-  (** [opt gen] shrinks towards [None] then towards shrinks of [gen]. *)
-  let opt ?(ratio : float = 0.85) (gen : 'a t) : 'a option t = fun st ->
+  let neg_int = int_neg
+
+  let option_ratio ~ratio (gen : 'a t) : 'a option t = fun st ->
```

Comment:
The interface in .mli doesn't use a label for `gen` AFAICS. I'm curious why this wasn't caught be the type-checker... 🤔
That however suggests that we should add a test for these new ones to ensure we don't roll out any broken signatures in the new interface... (it was when writing such tests I encountered the optional-argument issue)

Reply:
This is not a label for gen, it's just annotated with the type. But yes, the colors are tricky, I had to check in the source file ahah

Reply:
Sorry, brainfart on my end! 😅